cli/create.go

```diff
@@ -63,6 +63,7 @@ func (cc *CreateCommand) addFlags() {
 	flagSet.StringVar(&cc.utsMode, "uts", "", "UTS namespace to use")
 	flagSet.StringSliceVar(&cc.sysctls, "sysctl", nil, "Sysctl options")
 	flagSet.StringSliceVar(&cc.network, "net", nil, "Set networks to container")
+	flagSet.StringSliceVar(&cc.securityOpt, "security-opt", nil, "Security Options")
```

Comment:
I think the command line usage was not readable enough for users to understand even developers. I will profile an issue to fix this.